lib/keka.rb

```diff
@@ -31,6 +31,14 @@ def ok_if!(evaluator, msg = nil)
       end
     end
 
+    def err!(msg = nil)
```

Comment:
nit: are these names too close to the existing `Keka.ok` and `Keka.err`  methods? Normally ! and non-! methods with the same name are expected to be similar

Reply:
what do you recommend? i picked `ok!` and `err!` because they're consistent with `ok_if!`, `err_if!` etc. So maybe we should change `ok` and `err` - in fact i'm unsure if they should be public methods at all.

Reply:
`ok` and `err` have been quite handy in unit tests, whether that was intended usage or not. 
We could change those to something like `ok_obj` and `err_obj`? I agree that the consistency betweeen `ok!` and `ok_if!` is nice

Reply:
If we don't want to introduce a breaking change with this PR i'm ok deferring this conversation/update to a later time

Reply:
Renamed `.err` to .`err_result` and `.ok` to `.ok_result`